libs/wingsdk/src/fs/fs.ts

```diff
@@ -0,0 +1,279 @@
+import * as fs from "fs";
+import * as path from "path";
+import * as yaml from "yaml";
+import { InflightClient } from "../core";
+import { Json } from "../std";
+
+/**
+ * Convert the path to Windows format if running on Windows.
+ * @param p the path to convert
+ * @returns normalized path
+ */
+function localePath(p: string): string {
+  if (process.platform !== `win32`) {
+    return p;
+  }
+  return p.split(path.sep).join(path.win32.sep)
```

Comment:
it won't work on windows since in windows `path.sep === path.win32.sep` (https://nodejs.org/api/path.html#pathsep)
I think  this will work for all os:
`return p.split(path.posix.sep).join(path.sep)`